gtsfm/data_association/data_assoc.py

```diff
@@ -133,6 +128,7 @@ def run(
         ] / len(tracks_2d)
 
         # pick only the largest connected component
+        # TODO: remove this for hilti as disconnected components not an issue?
```

Comment:
not sure what you mean, disconnected components are an issue right?

Reply:
If we have priors between all IMU poses, even if there is are disconnected components at the data association step, a single BA can still work.

Reply:
Yeah, so we'd have to make that check configurable.

Reply:
Sounds good. Let me do that in a separate PR though.